app/views/crime_applications/index.html.erb

```diff
@@ -43,7 +45,7 @@
               <strong class="govuk-tag govuk-tag--blue">In progress</strong>
             </td>
             <td class="govuk-table__cell">
-              <%= button_to '#', method: :delete,
+              <%= button_to confirm_application_deletion_path(app.id), method: :get,
```

Comment:
I think `govuk-button` is adding quite a lot of margin-bottom, we have to add an override class:
`class: 'govuk-button govuk-button--secondary govuk-!-margin-bottom-0'`